### DISCUSSION TITLE
download failed

### DISCUSSION THREAD

**Discussion by ronanisgreat** [aclap-dev/video-downloadhelper#605] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://jav.guru/243207/mimk-111-after-the-war-ended-i-picked-up-a-beautiful-human-weapon-girl-with-big-breasts-and-brought-her-home-a-live-action-version-of-the-final-version-of-otonateikoku-kagyouno-meekichis-super/",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tls @ 0000022e3cd50200] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[tls @ 0000022e3d6b2fc0] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[tls @ 0000022e3d6cae80] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[tls @ 0000022e3d6cacc0] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[tls @ 0000022e3c945800] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[tls @ 0000022e3d68fa00] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[tls @ 0000022e3d5f5240] IO error: Error number -10054 occurred\r\n    Last message repeated 1 times\r\n[aac_adtstoasc @ 0000022e3cd6b240] Error parsing ADTS frame header!\r\n[mp4 @ 0000022e3db68c40] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 0000022e3cd7e140] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 0000022e3cedbb80] Error muxing a packet\r\n",
    "id": "coapp_failure_a5681505-113a-4c84-87d8-61a079ffc276",
    "downloadable_id": "downloadable_7113312596346517"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\nirma\\Downloads",
    "concurrent_downloads_max": 2,
    "show_success_notification_for_icognito": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 8
}

**Comment by paulrouget** [comment]
Thank you for the report!

Note: that's a ff failure with `Error parsing ADTS `.

**Comment by GrampaWildWilly** [comment]
I went to the web page & VDH was recognizing nothing.  I tried I think the first 3 servers identified under the player before I got something that would actually play in the web page.  That finally triggered the appearance of one choice in the VDH menu.  I launched the download & it's still running.

How long will this take?  I can't say.  The VDH progress status now tells me only the download speed & a time counter which appears to be telling me how long this download has been running.  More useful would be an indication of the size the downloaded file has grown to so far, & an estimated percentage complete so far.  These are items that used to be reported in previous releases of VDH.  It sure would be nice to get those back.

Meanwhile, I trawled through the Network Monitor.  Filtering on mpd gave nothing.  Filtering on m3u8 gave 3 results.  One was apparently a GIF, but the Type column for it was blank, & hovering the mouse over it did not display an image, which is what happens for proper images.  The other 2 hits on m3u8 were of Type HTML.  The file names were something.m3u8, but they were allegedly HTML.  That is pathological.  I ran their URLs through ffprobe & got 403 Forbidden.  Not invalid content, but rather not permitted.  There was nothing of the proper Type: x-mpegurl.  I don't know how you are downloading anything here.  There is no media content displayed in the Network Monitor, no mp4, no mkv, no m2ts, nothing.

Anyway, this thing is apparently 2 hours long.  I think it's going to take at least that long for VDH to complete this download.  I am not optimistic that it will end up being anything usable, but I will let it run & give it a chance to prove me wrong.  I'll post again after it completes, possibly long after it completes.

**Reply by GrampaWildWilly** [reply]
Oh.  Forgot to mention.  There was one thing in there of Type vtt.  I can assure you it was not WebVTT subtitles.  I was not surprised.

**Reply by GrampaWildWilly** [reply]
It took a bit under an hour & a half:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/2d49ffb6-4e83-4cbf-b334-2a77cb467ba7)

It played fine in VLC, although I only sampled it.  It seemed to have both video & audio right to the end.

I am running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.  This configuration was able to download this content, although it took a painfully long time.
